Makefile

```diff
@@ -23,10 +23,57 @@ $(MAKEFILE):
 
 -include $(MAKEFILE)
 
+# Conf vars
+GITHUB_API_KEY ?=
+LOGS_PATH ?= /var/log/docsrv
+LANDING_PATH ?= $(SHARED_PATH)/landing
+INIT_PATH ?= $(shell pwd)/../ci/docs/init.d
+CONF_PATH ?= $(shell pwd)/../ci/docs/conf.d
+
+#vars
+docker_container_name = docsrv-instance
+docker_image_name = docsrv-image
+
 #tools
 mkdir := mkdir -p
 goBuild := CGO_ENABLED=0 go build -o
+remove := rm -rf
```

Comment:
Lowecase variables? We are using uppercase and underscores for these variables in most other projects.